scripts/broad_dsde_workflows/GenotypeGVCFsScatterWf_170204.wdl

```diff
@@ -0,0 +1,173 @@
+## Copyright Broad Institute, 2017
+## 
+## This WDL workflow runs GenotypeGVCFs on a set of GVCFs to joint-call multiple 
+## samples, scattered across intervals.
+##
+## Requirements/expectations :
+## - GVCFs produced by HaplotypeCaller in GVCF mode 
+##
+## Outputs :
+## - A (typically but not necessarily multisample) VCF file and its index
+##
+## Cromwell version support 
+## - Successfully tested on v24
+## - Does not work on versions < v23 due to output syntax
+##
+## Runtime parameters are optimized for Broad's Google Cloud Platform implementation. 
+## For program versions, see docker containers. 
+##
+## LICENSING : 
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in 
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may 
```

Comment:
Actually my "correct" comment was an oversimplification. I should have clarified that I use these terms to distinguish two levels: programs are GATK, Picard, BWA, Cromwell, and tools, such as HaplotypeCaller, are what we invoke in programs that are toolkits, like GATK or Picard. So they're not interchangeable.